src/renderer/components/ExportCard.tsx

```diff
@@ -9,18 +10,38 @@ const Card = styled(Stack)({
   gap: '6px',
   backgroundColor: colors.grey[600],
   borderRadius: '5px',
+  position: 'relative',
+});
+
+const IconButton = styled(Stack)({
+  position: 'absolute',
+  right: '8px',
+  top: '8px',
 });
 
 interface Props {
+  onClose: () => void;
   progress: number;
 }
 
-const ExportCard = ({ progress }: Props) => {
+const ExportCard = ({ onClose, progress }: Props) => {
+  const exportText = progress === 1 ? 'Export Complete' : 'Exporting...';
+  const closeButton =
+    progress === 1 ? (
+      <IconButton
```

Comment:
Mui has a component called IconButton which you can use. Sorry for not mentioning earlier 🤣 
https://mui.com/material-ui/react-button/
https://mui.com/material-ui/api/icon-button/

Reply:
It returns a `IconButton` when progress reaches 1, because we don't want it during the exporting